ldb/export.py

```diff
@@ -292,7 +293,7 @@ def getname(obj):
                     name = obj.get("organization__name_prefix", "")
                     name += " "
                     name += obj.get("organization__name")
-                    name = re.sub("\s+", " ", name)
+                    name = re.sub(" +", " ", name)
```

Comment:
Hmmm... this changes the behaviour 🤔 Is that intended?

Reply:
That sounds strange. The python regex module doesn't accept regex anymore 🙈. According to the [documentation](https://docs.python.org/3/library/re.html) it should accept regex (also a `\s` in the example (however, the string there has a r before it, making it a raw string, so maybe just change it to `r"\s+"`)))

I might imagine someone copying a tab into the name field by exident. (But do agree that probably won't happen very often, so changing it to ` +` probably will be fine 🙈 )